alcs-frontend/src/app/features/board/dialogs/covenant/covenant-dialog.component.scss

```diff
@@ -59,34 +48,31 @@
 .board-menu-item {
   display: grid;
   grid-template-columns: 22px 1fr 22px;
+  width: 100%;
 }
 
-.favourite-board-icon-container {
-  width: 22px;
-  display: inline-block;
+::ng-deep .mat-mdc-menu-item .mdc-list-item__primary-text {
```

Comment:
Since this doesn't have host, in theory you would only need this once somewhere

Reply:
for some reason host breaks it and the style is not applied